pulp_2to3_migration/app/migrations/0003_modularity_yummetadatafile_erratum.py

```diff
@@ -0,0 +1,139 @@
+# Generated by Django 2.2.11 on 2020-03-23 16:33
```

Comment:
i would rename the migration to 0003_modularity_metafile_erratum_rpm ( to capture rpm models also..)
not critical, just suggestion